tests/BasicSearchTests.FunctionalTests.Core/EnvironmentSettings.cs

```diff
@@ -47,6 +50,45 @@ public static string IndexBaseUrl
             }
         }
 
+        public static string AzureSearchIndexUrl
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_azureSearchIndexUrl))
+                {
+                    _azureSearchIndexUrl = GetEnvironmentVariable("AzureSearchIndexUrl", "https://nuget-int-usnc.search.windows.net");
```

Comment:
Can we throw if the variable is not defined? Defaulting to INT seems pretty odd.

Reply:
Talked offline. This is the pattern used by other settings, I will keep this setting consistent.